shared/src/commonMain/kotlin/io.projectnewm.shared/login/Users.kt

```diff
@@ -0,0 +1,13 @@
+package io.projectnewm.shared.login
+
+data class LoginUser(val email: String, val password: String)
```

Comment:
nit: login is a noun, log in is an action

```suggestion
data class LogInUser(val email: String, val password: String)
```